releasenotes/notes/0.25/add-pauli-equivalences-74c635ec5c23ee33.yaml

```diff
@@ -1,5 +1,5 @@
 ---
-features:
+features_transpiler:
   - |
     The transpiler's built-in :class:`.EquivalenceLibrary` has been taught the circular Pauli
```

Comment:
I might mention `qiskit.circuit.equivalence_library.SessionEquivalenceLibrary` here as that's the concrete object used here. (it's not in the docs, there's nothing to really document there, so just mentioning the object path)